app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/account/components/draft_list_item.scala.html

```diff
@@ -74,7 +75,17 @@
             })
 }
 
-@dueDate = @{completionDate._1.map(d => d.plusDays(30L))}
+@dueDate = @{
```

Comment:
Its nearly always better to put this kind of logic (which is calculating a value) outside of a view

Reply:
Yes I too agree with you. As its temporary solution and we may remove it later, I feel updating a view is a quick fix.